x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/benchmark.ts

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
If the test suites are the same, what do you think about having these files soft-linked to this folder instead of duplicating them? would that work? wdyt about this approach?

Reply:
tbh I'm hesitant to investigate this route as:
1. it's not something that is recommended by the platform team, they are quite specific about copying existing tests over and adopting them for serverless.
2. Test cases I moved have a small difference in an additional header required, there are also tests that require more changes. Even if we make softlink/symlink approach work, it won't work for all tests cases, plus it might break at any point in time, as running the same tests is not the supported way, at least yet
3. The platform team is aware of the issue and there are some attempts to remove this duplication, eg. https://github.com/elastic/kibana/pull/161574 . 

Overall, I'd follow what's officially supported way, for now, the amount of tests we have don't make the duplication a show stopper in my opinion